docs/commands/lint.md

```diff
@@ -1,16 +1,12 @@
 # `lint`
 
-## Introduction
+## Why you need it
 
-Redocly OpenAPI CLI can identify and report on problems found in OpenAPI definitions. This helps you avoid bugs and make API definitions more consistent.
+One of the most powerful features of Redocly OpenAPI CLI is its ability to identify problems found in OpenAPI definition files. This is achieved through the `lint` command, which helps your definitions conform to a consistent set of rules.
 
-The `lint` command reports on problems and executes preprocessors and rules. Unlike the `bundle` command, `lint` doesn't execute decorators.
+The `lint` command ensures that your definition file structure is valid (according to the OpenAPI Specification and your own custom rules, if any) and contains no errors. It also executes preprocessors and rules (unlike the `bundle` command, `lint` doesn't execute decorators). To learn more about preprocessors and rules, see the [custom rules](../resources/custom-rules.md) page.
 
-:::success Tip
-To learn more about preprocessors and rules, refer to the [custom rules](../resources/custom-rules.md) page.
-:::
-
-## Usage
+Always the over-achiever, `lint` has options which you can execute alone, or combine with others:
```

Comment:
issue: We list some of the options here, but not all of them. Why? Also, again - removing the Usage/Options headings makes the page structure inconsistent with other commands pages.

Reply:
suggestion: Maybe we can drop this `Usage` section and these options altogether, but keep the `Options` section with all supported options listed in one place? That way, we won't have this redundant/incomplete situation where some options are mentioned twice, and some just once. And then we can rename the `Examples` section to `Usage examples`. That structure won't be too drastically different from all the other pages (it's easy to make that change across all pages), and it will still have the effect of cleaning up the page. What do you think?

Reply:
To answer your question, I don't think the Usage section on the current page is meant to show an example of all commands, just a few, including combos. I will list all of them in the one place in the new page and show single commands as well as combos.

Re your suggestion - will do.